conf/application.conf

```diff
@@ -94,7 +94,7 @@ microservice {
       enrolment-store-proxy {
         protocol = http
         host = localhost
-        port = 7775
+        port = 9807
```

Comment:
Should this be changing to `9807`? This will break the pipeline for the other journey tests if merged in.